source/iterators.tex

```diff
@@ -4406,7 +4406,7 @@
 \begin{itemdescr}
 \pnum
 \constraints
-The expression \tcode{x + n} shall be valid and have type \tcode{Iterator}.
+The expression \tcode{x.base() + n} shall be valid and have type \tcode{Iterator}.
 
 \pnum
 \returns \tcode{x + n}.
```

Comment:
This change creates an inconsistency between the constraints and the returned value.

`move_iterator:::operator+(difference_type)` has no constraints, so the expression in question is always valid (even though instantiation of the operator might fail), so presumably we're missing a constraint around [move.iter.nav]p7? Whether or not this expression is SFINAEable is not editorial, and the current specification appears to be normatively wrong, so I think this part needs to go to LWG.

Reply:
http://wg21.link/LWG3293 attempts to solve this.

Reply:
Thanks for the reminder that I need to fix this issue's P/R.